modules/network-observability-viewing-alerts.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * network_observability/network-observability-operator-monitoring.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-alert-dashboard_{context}"]
+= Viewing the alert dashboard
+
+You can access alerts for the Network Observability Operator dashboard in the *Alerting* page in the web console.
+
+.Prerequisites
+
+* You have the Network Observability Operator installed.
+//* You have access to the cluster as a user with the `cluster-admin` role or with view permissions for all projects. --question-is this feature for admins only?
+
+.Procedure
+
+. From the *Administrator* perspective in the web console, navigate to *Observe* → *Alerting*.
+
+. From the *Alerts* list, select the *NetObservLokiError* alert.
+//should the name of this alert be NetworkObservabilityLokiHealthError or just LokiHealthError? AFAIK we don't usually use NetObserv downstream.  
```

Comment:
We can try to see if is can be renamed for downstream, but to be honest, I'm not sure we will be able to translate NetObserv to Network Observability everywhere it may appear, without too much hassle. Maybe keep it as `NetObservLokiError` at the moment and we will open a bug to see if we can rename for downstream.
As another example, the default namespace is still netobserv. Some things will be hard to change.